workers/github_worker_base.rb

```diff
@@ -27,6 +27,7 @@ def initialize(provider_credential: nil, project: nil)
       self.project = project
 
       self.target_branches_set = Set.new
+      project.job_triggers = [] # WARN: DELETE
```

Comment:
TODO: 😄 We search for these, but not # WARN

Reply:
Those WARN are for myself. There're a lot of TODO, so I use WARN to find code I have to delete before merge 😅